data/json/snippets/effect_on_conditions.json

```diff
@@ -189,84 +189,84 @@
       "a gentle voice saying \"The place you called home is not there anymore.  Come with me and find your new home.\"",
       "a gentle voice saying \"What good comes from hiding from the truth in a mausoleum of brick walls?\"",
       "a gentle voice saying \"There is no amount of concrete or stone you could put between you and the veil that is thinner than a shadow.\"",
-      "a gentle voice saying \"You want to sleep now?  Just moments after you have awaken us from our eternal slumber?\"",
+      "a gentle voice saying \"You want to sleep now?  Just moments after you finally wake up?\"",
```

Comment:
This seems to change the meaning and sounds just slightly off to me.  Maybe "Just moments after you have finally awoken? instead?